poetry.lock

```diff
@@ -295,7 +295,7 @@ python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
 
 [[package]]
 name = "google-auth"
```

Comment:
You can't specify dev extras. But I agree it may be easier to just make them dev deps. But let's table that for another PR, I'm going to revert this change here anyways.

Reply:
Filed #212.